iOS Client/DevotionsController/DevotionDetailViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  DevotionDetailViewController.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 11/15/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import UIKit
+import ReSwift
+
+class DevotionDetailViewController: UIViewController, StoreSubscriber {
+
+    @IBOutlet weak var monthLabel: UILabel!
+    @IBOutlet weak var dayLabel: UILabel!
+    @IBOutlet weak var verseLabel: UILabel!
+    @IBOutlet weak var verseTextLabel: UILabel!
+    @IBOutlet weak var bodyLabel: UILabel!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureBackButton()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        MBStore.sharedStore.subscribe(self)
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        MBStore.sharedStore.unsubscribe(self)
+    }
+
+    func newState(state: MBAppState) {
+        if let devotion = state.devotionState.selectedDevotion,
+            let date = Formatters.devotionDateFormatter.date(from: devotion.date),
+            let monthInt = Formatters.calendar?.component(.month, from: date),
+            let day = Formatters.calendar?.component(.day, from: date) {
+            monthLabel.text = Formatters.getMonth(fromInt: monthInt)
+            dayLabel.text = String(describing: day)
```

Comment:
that's cool, never seen String describing before

Reply:
maybe worth saving the formatted month string and the formatted day string as lazy properties on the devotion class so we don't have to burden this function with these details